workshop/README.md

```diff
@@ -162,7 +162,7 @@ IBM Cloud provides the capability to run applications in containers on Kubernete
 
 [Lab 0](Lab0) (Optional): Provides a walkthrough for installing IBM Cloud command-line tools and the Kubernetes CLI. You can skip this lab if you have the IBM Cloud CLI, the container-service plugin, the containers-registry plugin, and the kubectl CLI already installed on your machine.
 
-[Lab 1](Lab1): This lab walks through creating and deploying a simple "hello world" app in Node.JS, then accessing that app.
+[Lab 1](Lab1): This lab walks through creating and deploying a simple "guestbook" app written in Go as a net/http Server([negroni](https://github.com/codegangsta/negroni) based) then accessing that app.
```

Comment:
we don't need this link

Reply:
+1 people won't care what http package we're using